cmd/podman/load.go

```diff
@@ -64,10 +65,10 @@ func loadCmd(c *cliconfig.LoadValues) error {
 	if runtime.Remote && len(input) == 0 {
 		return errors.New("the remote client requires you to load via -i and a tarball")
 	}
-	if len(input) == 0 {
-		input = "/dev/stdin"
-		c.Input = input
-
+	if input == "" {
+		if terminal.IsTerminal(int(os.Stdin.Fd())) {
+			return errors.Errorf("can not use terminal, redirect the archive or using the --input flag required")
```

Comment:
I'd like @edsantiago to take a look at this line, the bit after the 'or' needs help.  I'd suggest "or use the --input option as required"

Reply:
I'm fine with Ed's suggestion except 'flag' s/b 'optoin'.  If you pull up the man page, you'll find --input listed under "options" not "flags"